Troubeshooting.md

```diff
@@ -39,3 +39,10 @@
 
 2. If the command `pod install` is outputting version mismatch errors for `KlaviyoSwift`, please run `pod update KlaviyoSwift`
    as indicated in the error message to update your local pods spec repo.
+
+### `UNUserNotificationCenter` delegate methods not being called
+
+If you are not seeing the delegate methods for `UNUserNotificationCenter` being called in `AppDelegate`, there are two possible reasons for this,
+
+1. [Notifee](https://notifee.app/) intercepts the AppDelegate delegate methods and hence you may not receive the delegate calls if notifee is included in the iOS project. The solution is to remove notifee dependency from your project or exclude it for iOS.
```

Comment:
So does notifee also swizzle? But not provide a way to turn it off?

Reply:
discussed offline.